consensus/istanbul/validator/default_test.go

```diff
@@ -210,6 +212,8 @@ func TestStickyProposer(t *testing.T) {
 }
 
 func TestDefaultSet_SubList(t *testing.T) {
+	fork.SetHardForkBlockNumberConfig(&params.ChainConfig{IstanbulCompatibleBlock: istanbul.TestBlockHF})
```

Comment:
@yoomee1313 Could you change back the HardForkBlockNumberConfig with `defer`? This change may affect other test cases.

Reply:
@aidan-kwon wow.. thanks for good idea. I also worried about the problem that the chainConfig was shared between the tests.

Reply:
Uhm... It may not work well. `SetHardForkBlockNumberConfig` calls `once.Do()` internally. Does it need to be called once?

Reply:
Yes, once.Do() needs to be removed.. I reconsidered about using it and decided to remove once.Do() because there's already a check logic whether the global config is set or not. It's been deleted and reset the global config at `defer`, so please take a look. By the way, I think the change is too big.. What do you think?